sdk/file_sdk/Common/file_format.cpp

```diff
@@ -0,0 +1,25 @@
+#include "file_format.h"
+
+namespace mip {
+namespace file {
+
+FileFormat::FileFormat(
+    std::shared_ptr<IStream> file,
+    const string& extension)
+  : mFile(file),
+    mExtension(extension)

```

Comment:
init with what? its a vector

---
In reply to: [134435631](https://github.com/AzureAD/rms-sdk-for-cpp/pull/133#discussion_r134435631) [](ancestors = 134435631)